docker/publish-snapshot.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+IMAGE="hivemq/hivemq-ce:snapshot"
+
+cd "$(dirname $0)/../"
+HIVEMQ_VERSION=$(./gradlew properties | grep ^version: | sed -e "s/version: //")
+echo "Building Docker image for HiveMQ ${HIVEMQ_VERSION}"
+./gradlew hivemqZip
+cd docker
+cp ../build/zip/hivemq-ce-${HIVEMQ_VERSION}.zip .
+docker build --build-arg HIVEMQ_VERSION=${HIVEMQ_VERSION} -f Dockerfile -t ${IMAGE} .
+rm -f hivemq-ce-${HIVEMQ_VERSION}.zip
+
+if [[ ${PUSH_IMAGE} == true ]]; then
+    echo "Pushing image"
+    docker push ${IMAGE}
+fi
```

Comment:
This file is now basically the same as build.sh + the additional if statement, we could just move the if to build.sh to remove duplication